### ISSUE TITLE
Add support for static mappers

### ISSUE THREAD

**Issue by zitmen** [riok/mapperly#58]
Hi, would you consider support of static mappers?
For example this:
```csharp
[Mapper]
public static partial class DtoMapper
{
    public static partial CarDto CarToCarDto(Car car);
}
```
It currently generates an empty class.
Since I did not see this in the examples and don't have experience with MapStruct, I don't dare to say it is a bug :-)
Thanks!

**Comment by latonz** [comment]
Thank you for your feature request, sounds like a good idea to implement this. Feel free if you want to contribute a PR.

**Comment by TheR00st3r** [comment]
Static Classes with ExtensionMethods would be rly great

   ``` csharp
    [Mapper]
    public partial class DtoMapper
    {
        public static partial User ToUser(this ApplicationUser applicationUser);
    }
   ```

**Comment by latonz** [comment]
Implemented by #76
